shopfloor_packing_info/readme/DESCRIPTION.rst

```diff
@@ -1,3 +1,7 @@
+DEPRECATED: This module is deprecated in favor of stock_picking_partner_note,
+which is now used in shopfloor to display notes in the app.
```

Comment:
You could add in this module the dependency to `stock_picking_partner_note` and the migration script to move data to the new model